selfdrive/car/honda/carcontroller.py

```diff
@@ -99,6 +99,12 @@ def process_hud_alert(hud_alert):
                       "lanes_visible", "fcw", "acc_alert", "steer_required"])
 
 
+def rate_limit_steer(new_steer, last_steer):
```

Comment:
You can use the current rate limiting code, and just pas in 0 user/eps torque so we don't duplicate code